test/form-builder/TextField.test.tsx

```diff
@@ -1,17 +1,83 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/no-empty-function */
+
 import React from 'react';
 import TextField from '../../src/form-builder/TextField';
+import { waitFor } from '@testing-library/react';
+
+import { buildRenderForm, changeValue } from '../utils';
 
-import { buildRenderForm } from '../utils';
+const renderForm = buildRenderForm();
 
-const renderForm = buildRenderForm({ name: 'asdf' });
+const getInput = (container: HTMLElement) => {
+  const input = container.querySelector('va-text-input') as HTMLInputElement;
```

Comment:
:thinking: Have you tried to cast this as `HTMLVaTextInputElement` instead? I'd like to avoid "downgrading" the component if we can.

This type should live in `web-components/dist/types/components.d.ts`

Reply:
Yeah, I'm with you. I tried, but I couldn't figure out how to.

I can see the type in there, but I can't import from a `.d.ts` file. The interface is declared in a `declare global` block, so theoretically if I could get TypeScript to see that, I'd be set, but I don't know how to make it happen. :sigh:

Any ideas?

Reply:
:thinking: Nothing immediate comes to mind, but I'll dabble a bit with the select field.

Reply:
I expect this should be less of a problem when we are able to `import` the React bindings. :man_shrugging: